rosidl_generator_py/resource/_msg.py.em

```diff
@@ -87,6 +87,10 @@ for member in message.structure.members:
 @[end if]@
 @#>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
+from os import getenv
+
+ros_python_check_fields = getenv("ROS_PYTHON_CHECK_FIELDS")
```

Comment:
I think it is worthwhile to put a comment here saying that you are doing this at the module level since it was too slow at the per-object level.

Reply:
Done. Take a look and let me know if the comment is fine.